city_scrapers/spiders/chi_ssa_28.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa28Spider(CityScrapersSpider):
+    name = "chi_ssa_28"
+    agency = "Chicago Special Service Area #28 Six Corners"
+    timezone = "America/Chicago"
+    start_urls = ["https://sixcorners.com/ssa28"]
+    location = {"name": "Portage Arts Lofts", "address": "4041 N. Milwaukee Ave. #302"}
```

Comment:
The address part should include "Chicago, IL 60641" since we're adding it statically